sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 3.0.3 (2020-11-10)
```

Comment:
seems like `Azure:release/ai-formrecognizer_3.0.3` is based on 3.0.0?

Reply:
Yes, to correct 3.0.2 and 3.0.1 had changes targeting the preview version of the service.
https://github.com/Azure/azure-sdk-for-java/blob/azure-ai-formrecognizer_3.0.2/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/FormRecognizerClientImpl.java#L143